### PR TITLE
Connection class added

### PR DESCRIPTION
**Pull request by bizley**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | https://github.com/yiisoft/yii2/pull/18560

This fixes regression introduced in yiisoft/yii2#18476
Tests are expected to fail.

### CODE REVIEW

**Comment by bizley** [comment]
I'm not sure why the tests are not failing :D

**Comment by bizley** [comment]
Oh, this connection extends `yii\db\Connection` :grin: Everything is fine, tests should succeed. Anyway, still ok to add new property. And tests should be more reliable now.

**Comment by bizley** [comment]
Changes removed except the ones made to the tests setup.

File: tests/TestCase.php [hunk 1]
```diff
@@ -41,16 +41,16 @@ class TestCase extends \PHPUnit\Framework\TestCase
     protected function setUp()
     {
         parent::setUp();
-        if (!extension_loaded('pdo') || !extension_loaded('pdo_mysql')) {
```
Comment (samdark) [inline tests/TestCase.php:44]: The check is needed since MySQL client is used to connect to Sphinx.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**